lib/Analysis/Allocation.cpp

```diff
@@ -125,11 +125,24 @@ class AllocationAnalysis {
     // TODO(Keren): Add convert ops
     if (auto reduceOp = dyn_cast<triton::ReduceOp>(op)) {
       // TODO(Keren): Reduce with index is not supported yet.
+      unsigned axis = reduceOp.axis();
```

Comment:
Not sure whether it is possible to reduce across multiple axis？

Reply:
It's not allowed in triton